lambdas/db-migration/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import Knex from 'knex';
+import * as path from 'path';
+
+export type Command = 'latest';
+
+export interface HandlerEvent {
+  command?: Command,
+  env?: NodeJS.ProcessEnv
+}
+
+const getRequiredEnvVar = (name: string, env: NodeJS.ProcessEnv): string => {
+  const value = env?.[name];
+
+  if (value) return value;
+
+  throw new Error(`The ${name} environment variable must be set`);
+};
+
+const getConnectionConfig = (env: NodeJS.ProcessEnv): Knex.PgConnectionConfig => ({
```

Comment:
To me, this function seems like a great candidate for the `db` package if we're going to have one

Reply:
Same argument I made earlier, the config is going to be coming from different places depending on the situation, so I don't think it should be in a library.

Reply:
Hmm okay. Both of the current invocations seem to be pulling from the env vars, hence the suggestion